helm/rook-operator/values.yaml

```diff
@@ -414,12 +414,12 @@ global:
   # global toggle to enable/disable creation of all resources.
   rookEnabled: true
 
+  # configuration for the rook-cephc-cluster subchart.
+  rookCephCluster:
+    toolbox:
+      # YAML anchor which references main rook version value earlier in this file.
+      version: *rookVersion
+
```

Comment:
this needed to be moved to the globals section as it wasn't accessible in the subchart (which is where it is used)